edx_proctoring/models.py

```diff
@@ -173,11 +177,14 @@ class ProctoredExamStudentAttemptStatus(object):
 
     # status alias for sending email
     status_alias_mapping = {
-        submitted: _('pending'),
-        verified: _('satisfactory'),
-        rejected: _('unsatisfactory')
+        submitted: ugettext_noop('pending'),
+        verified: ugettext_noop('satisfactory'),
+        rejected: ugettext_noop('unsatisfactory')
     }
 
+    def __unicode__(self):
```

Comment:
I added these because there was a PEP8 failure for each model that didn't have a ``__unicode__`` method. The code in edx-platform seems to be very inconsistent about which one to use. 

It seems that probably ``__repr__`` is the better choice, so I'll try converting these and see what happens:

http://stackoverflow.com/questions/1984162/purpose-of-pythons-repr